packages/common/src/paginator.js

```diff
@@ -34,48 +34,70 @@ var util = require('./util.js');
 
 /*! Developer Documentation
  *
- * streamRouter is used to extend `nextQuery`+callback methods with stream
- * functionality.
+ * paginator is used to auto paginated `nextQuery` methods as well as
```

Comment:
Just a bump that I'm just saying this should be "paginator is used to auto-paginate `nextQuery` methods", in place of "paginator is used to auto paginated `nextQuery` methods"